R/utils-validation.R

```diff
@@ -1,13 +1,13 @@
 valid_types <- function() {
   # TODO: Update when we have a more fixed list
   # And hopefully more CDISC compliant
-  c("predecessor", "derivation", "row")
+  c("predecessor", "derivation", "row", "assigned")
 }
 
 assert_type <- function(type) {
   types <- valid_types()
   if (!type %in% types) {
-    cli::cli_abort("@type must be one of {.val {types}}")
+    cli::cli_abort("@type must be one of {.val {types}}. Found {.code type}")
```

Comment:
The error message is not displaying the actual invalid type value. The variable `type` should be interpolated using curly braces: `{.code {type}}`
```suggestion
    cli::cli_abort("@type must be one of {.val {types}}. Found {.code {type}}")
```